source/introduction.md

```diff
@@ -16,44 +21,57 @@ quadrupole, or a diffracting crystal. A lattice element may define a region in s
 distinguished by the presence of (possibly time-varying) electromagnetic fields,
 materials, apertures and other possible engineered structures. However, lattice elements
 are not restricted to being something physical and may, for example, just mark a particular point in space
-(EG: **Marker** elements), or may designate where beam lines intersect (**Fork** elements).
+(EG: `Marker` elements), or may designate where beam lines intersect (`Fork` elements).
 By convention, element names in PALS will be upper camel case.
 
-
+%---------------------------------------------------------------------------------------------------
+(s:branches)=
 ## Lattice Branches
 
-A lattice **branch** holds a collection of lattice elements. 
-There are two types of branches. One type, called a "tracking branch", holds
-an ordered array of lattice elements that gives a
-sequence of elements to be tracked through. A tracking branch can represent something like a
-storage ring, transfer line or Linac.
+A lattice `branch` holds a collection of lattice elements which is an ordered array of 
+lattice elements that gives the sequence of elements to be tracked through. 
+A branch can represent something like a storage ring, transfer line or Linac.
```

Comment:
Add comma between 'elements' and 'which' above.
Replace 'simplist' with 'simplest' below.
Also, add comma between overlap and which.

Reply:
Have reworded this.